packages/backend/src/utils/modelsUtils.ts

```diff
@@ -20,7 +20,7 @@ import { join, posix } from 'node:path';
 import { getPodmanCli } from './podman';
 import { process } from '@podman-desktop/api';
 
-export const MACHINE_BASE_FOLDER = '/home/user/ai-lab/models/';
+export const MACHINE_BASE_FOLDER = '/home/user/ai-lab/models';
```

Comment:
This is necessary to remove the trailing slash, to compare with `ModelInfo#file#path` later without having to use magic trick